### PR TITLE
[BEAM-13880] [Playground] Increase test coverage for the setup_tools package

### PR DESCRIPTION
**Pull request by pavel-avilov**
[[BEAM-13880]](https://issues.apache.org/jira/browse/BEAM-13880)
Add unit tests to increase test coverage for the setup_tools package.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by pavel-avilov** [comment]
@ilya-kozyrev @KhaninArtur

File: playground/backend/internal/setup_tools/builder/setup_builder_test.go [hunk 1]
```diff
@@ -195,10 +231,10 @@ func TestCompiler(t *testing.T) {
 		{
 			// Test case with calling Setup with correct data.
 			// As a result, want to receive an expected compiler builder.
-			name: "Test correct compiler builder",
+			name: "Test correct compiler builder with java sdk",
```
Comment (KhaninArtur) [inline playground/backend/internal/setup_tools/builder/setup_builder_test.go:234]: Should we do the same test for Go and SCIO?

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: playground/backend/internal/setup_tools/builder/setup_builder_test.go [hunk 2]
```diff
@@ -76,18 +112,18 @@ func TestValidator(t *testing.T) {
 			// As a result, want to receive an expected validator builder.
 			name: "Test correct validator builder",
 			args: args{
-				paths:  paths,
-				sdkEnv: sdkEnv,
+				paths:  pythonPaths,
+				sdkEnv: pythonSdkEnv,
```
[the remark this answers is not in the document]
Comment (pavel-avilov) [reply playground/backend/internal/setup_tools/builder/setup_builder_test.go:116]: Done

File: playground/backend/internal/setup_tools/builder/setup_builder_test.go [hunk 3]
```diff
@@ -143,16 +179,16 @@ func TestPreparer(t *testing.T) {
...
 			want:    nil,
 			wantErr: true,
 		},
 		{
 			// Test case with calling Setup with correct SDK.
 			// As a result, want to receive an expected preparer builder.
-			name:    "correct sdk",
-			args:    args{*paths, pipelineOptions, sdkEnv, &validationResults},
+			name:    "Correct sdk",
+			args:    args{*pythonPaths, pipelineOptions, pythonSdkEnv, &validationResults},
```
[the remark this answers is not in the document]
Comment (pavel-avilov) [reply playground/backend/internal/setup_tools/builder/setup_builder_test.go:191]: Done

File: playground/backend/internal/setup_tools/builder/setup_builder_test.go (hunk 1 above)
Comment (pavel-avilov) [reply playground/backend/internal/setup_tools/builder/setup_builder_test.go:234]: Done

File: playground/backend/internal/setup_tools/builder/setup_builder_test.go [hunk 4]
```diff
@@ -271,27 +403,49 @@ func TestTestRunner(t *testing.T) {
 		want *executors.ExecutorBuilder
 	}{
 		{
-			// Test case with calling Setup with correct data.
-			// As a result, want to receive an expected test builder.
-			name: "Test correct test builder",
+			name: "Test correct run builder with Java sdk",
```
[the remark this answers is not in the document]
Comment (pavel-avilov) [reply playground/backend/internal/setup_tools/builder/setup_builder_test.go:406]: Done

**Review by KhaninArtur — APPROVED** [review]
LGTM

**Review by daria-malkova — APPROVED** [review]
LGTM

**Comment by pabloem** [comment]
lgtm merging

**Pull request merged**